packages/ts/react-crud/src/autocrud.obj.css

```diff
@@ -1,6 +1,7 @@
 .auto-crud {
   display: flex;
   overflow: hidden;
+  position: relative;
```

Comment:
Not sure where this issue came from, but without this the box shadow of the form area expands outside of the `AutoCrud`:

<img width="901" height="607" alt="Bildschirmfoto 2025-09-04 um 10 31 24" src="https://github.com/user-attachments/assets/48b99f87-c41c-47b7-9609-3e22016b382f" />